lib/auth/resetpasswordtoken.go

```diff
@@ -129,6 +129,41 @@ func (s *AuthServer) CreateResetPasswordToken(ctx context.Context, req CreateRes
 	return s.GetResetPasswordToken(ctx, token.GetName())
 }
 
+// proxyDomainGetter is a reduced subset of the Auth API for formatAccountName.
+type proxyDomainGetter interface {
+	GetProxies() ([]services.Server, error)
+	GetDomainName() (string, error)
+}
+
+// formatAccountName builds the account name to display in OTP applications.
+// Format for accountName is user@address. Use is passed in, this function
+// tries to find the best available address.
+func formatAccountName(s proxyDomainGetter, username string, authHostname string) (string, error) {
```

Comment:
you can collapse `username string, authHostname string` to `username, authHostname string`